testing/a11y/src/Component.cy.tsx

```diff
@@ -6,6 +6,34 @@ import * as React from 'react';
 import * as allExamples from 'examples';
 import { ThemeProvider } from '@itwin/itwinui-react';
 
+// Axe config rules docs: https://www.deque.com/axe/core-documentation/api-documentation/#api-name-axeconfigure
+// Axe rules that apply to multiple examples
+type commonRulesKeys = 'ignore_color_contrast_for_select_placeholder';
```

Comment:
I added this union `type` to store all the keys for `commonRules` so that if we do `commonRules.key_does_not_exist`, it gives a TS error during development time, instead of returning `undefined` at runtime. Is there a better way to do this?